src/modules/tasks.js

```diff
@@ -0,0 +1,211 @@
+    <button type='button' class='confirmTask btn btn-secondary' id='confirmTask'>Create task</button>
+    <button type='button' class='dismissTask btn btn-light m-3' id='dismissTask'>Dismiss</button>
+  `;
+
+    document.getElementById('datePicker').setAttribute('min', setDate());
+    countForm = true;
+  } else {
+    taskForm.innerHTML = '';
+    countForm = false;
+  }
+};
+
+const changeStatus = (index) => {
+  // eslint-disable-next-line
+  myTasks[index].finished ? myTasks[index].finished = false : myTasks[index].finished = true;
+};
+
+const deleteAllTasks = (currentProject) => {
+  const arr = myTasks.filter((task) => task.projectsN !== currentProject);
+  myTasks = arr;
+  first.innerHTML = '';
+  tableArticle.innerHTML = '';
+  todayContainer.innerHTML = '';
+};
+
+const makeEditForm = (task) => {
+  editTaskForm.innerHTML = `
+      <input type='text' class=' my-2' name='EdiTaskName' id='edit-task-name' value='${task.title}'>
+      <textarea  id='editComment' class='my-2' rows='3' maxlength='150'> ${task.comment}</textarea>
+      <select class='form-select me-4' id='selectPriority1' aria-label='Default select example'>
+              <option selected value='1'>Priority ${Flag(task.priority)}</option>
+              <option class='one' value='1'>&#xf024; Priority 1</option>
+              <option class='two' value='2'>&#xf024; Priority 2</option>
+              <option class='three' value='3'>&#xf024; Priority 3</option>
+              <option class='four' value='4'>&#xf024; Priority 4</option>
+      </select>
+      <input type="date" max='2110-13-13' id='datePicker1' value='${task.date}'>
+      <button type='submit' class='confirmEditTask btn btn-secondary' id='confirmEditTask'>Edit task</button>
+      <button type='button' class='dismissEditTask btn btn-light m-3' id='dismissEditTask'>Dismiss</button>
+    `;
+
+  document.getElementById('datePicker1').setAttribute('min', setDate());
+};
+
+const makeToday = () => {
+  todayContainer.innerHTML = '';
+  // eslint-disable-next-line
+  const todayTasks = myTasks.filter((task) => task.deadline === setDate() && task.finished === false);
+  todayTasks.forEach((task) => {
+    todayContainer.innerHTML += `
+  <div class="card mx-auto mt-3 bg-light">
+            <div class="card-body bg-light">
+              <h5 class="card-title">${task.title}</h5>
+              <p class="card-text">${task.comment}</p>
+            </div>
+            <ul class="list-group  bg-light list-group-flush">
+              <li class="list-group-item bg-light">Project: ${task.projectsN}</li>
+              <li class="list-group-item bg-light">Priority: ${Flag(task.priority)}</li>
+              <li class="list-group-item bg-light">Status: ${modalStatus(task.finished)}</li>
+              <li class="list-group-item bg-light">Deadline: ${task.deadline}</li>
+            </ul>
+
+          </div>`;
+  });
+};
+
+export {
+  // eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files